gradle.properties

```diff
@@ -7,7 +7,7 @@ base_group=com.ldtteam
 base_mod_id=structurize
 
 minecraft_version=1.16.4
-forge_version=35.0.1
+forge_version=35.1.7
```

Comment:
This is not the latest.
When we update, we should probably either go to the lastest recommended, or if that is not selected (like here) then we should go to the latest available version.

Reply:
pr was started before 35.1.10, sure can update